backend/tracim_backend/applications/share/lib.py

```diff
@@ -60,7 +60,7 @@ def share_content(
         do_notify: bool = False,
     ) -> typing.List[ContentShare]:
         content_shares = []
-        created = datetime.now()
+        created = datetime.utcnow()
```

Comment:
why do you use `utcnow()` now? What if the server is not configured in UTC?

Reply:
I do use utcnow() to have constant time mecanism. When checking for session, i do see a bug related to delta time calculation between utc based time and timezoned time (for reissue time which is deleted now), we do use utcnow() almost everywhere in code, and to prevent these issue, i do think we should always use utctime.

Reply:
so, what about deploying a tracim server on a non UTC configured server?

> When checking for session, i do see a bug related to delta time calculation between utc based time and timezoned time

What bug? If there is a bug in session mechanism, I'm not sure this is in the share library that we should fix it.

I do not see relationship between the description of bug #3091 and the bug you are explaining here. You should not fix bugs in other ones.

I feel like there are missing explanations for this MR to be accepted

Reply:
>  so, what about deploying a tracim server on a non UTC configured server? 

Don't sure to understand. utcnow give a constant value accross server whereas now is timezone dependent. Using utcnow make thing constant, so it should have a constant result now.

> What bug? If there is a bug in session mechanism.
It renew session each time because the delta time difference was always 2h.

>  I'm not sure this is in the share library that we should fix it.

I'm not sure but i do prefer to be constant in all tracim code about this to prevent issue related to this. Because i do see this issue about now() and utcnow(), i do check usage of now() in all tracim code to verify if we could have similar issue in another place.

Reply:
@inkhey, @grignards explained me what you talked  about this and what you decided. Your decision is ok for me.